example/adversary/README.md

```diff
@@ -1,7 +1,7 @@
 # Adversarial examples
 
 This demonstrates the concept of "adversarial examples" from [1] showing how to fool a well-trained CNN.
-The surprising idea is that one can easily generate examples which the CNN will consistently 
-make the wrong prediction for that a human can easily tell are correct.
+Adversarial examples are examples that humans can easily classify according to their true label but where the model is confused.
```

Comment:
Adversarial examples are samples where the input has been manipulated to confuse a model (i.e. confident in an incorrect prediction) but where the correct answer still appears obvious to a human.

This method for generating adversarial examples uses the gradient of the loss with respect to the input to craft the adversarial examples.